frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -240,3 +393,98 @@ function areBoundingBoxesValid(userBoundingBoxes: UserBoundingBox[]): {
   }
   return { valid: false, reason: "All bounding boxes must have the same size." };
 }
+
+function AnnotationsCsvInput({
+  onAdd,
+}: {
+  onAdd: (newItems: Array<AnnotationWithDataset<APIAnnotation>>) => void;
+}) {
+  const [value, setValue] = useState("");
+  const onClickAdd = async () => {
+    const newItems = [];
+
+    const lines = value.split("\n");
+    for (const line of lines) {
+      const [annotationUrlOrId] = line.trim().split(",");
+      if (annotationUrlOrId.includes("/")) {
+        newItems.push({
+          annotationId: annotationUrlOrId.split("/").at(-1) as string,
+        });
+      } else {
+        newItems.push({
+          annotationId: annotationUrlOrId,
+        });
+      }
+    }
+
+    const newAnnotationsWithDatasets = await Promise.all(
+      newItems.map(async (item) => {
+        const annotation = await getAnnotationInformation(item.annotationId);
+        const dataset = await getDataset({
+          owningOrganization: annotation.organization,
+          name: annotation.dataSetName,
+        });
+
+        const volumeTracings: ServerVolumeTracing[] = await Promise.all(
+          annotation.annotationLayers
+            .filter((layer) => layer.typ === "Skeleton")
+            .map(
+              (layer) =>
+                getTracingForAnnotationType(annotation, layer) as Promise<ServerVolumeTracing>,
+            ),
+        );
+
+        return {
+          annotation,
+          dataset,
+          volumeTracings,
+        };
+      }),
+    );
+
+    onAdd(newAnnotationsWithDatasets);
+  };
+  return (
+    <div>
+      <FormItem
+        name="annotationCsv"
+        label="Annotations CSV"
+        hasFeedback
+        initialValue={value}
+        rules={[
+          () => ({
+            validator: (_rule, value) => {
+              const valid = (value as string)
+                .split("\n")
+                .every((line) => !line.includes("#") && !line.includes(","));
+
+              return valid
+                ? Promise.resolve()
+                : Promise.reject(
+                    new Error(
+                      "Each line should only contain an annotation ID or URL (without # or ,)",
+                    ),
+                  );
+            },
+          }),
+        ]}
+      >
+        <TextArea
+          className="input-monospace"
+          placeholder="annotationUrlOrId[, colorLayerName, volumeLayerId]"
```

Comment:
See my feedback from testing

Reply:
I simply removed the option to directly enter `colorLayerName` or `volumeLayerId`. If you think this would be a nice improvement, I'd prefer to implement this as a follow-up. The current code version simply does not support the option to initially supply a color layer name and volume layer id.